src/main/java/de/tum/www1/orion/connector/client/ArtemisClientConnector.kt

```diff
@@ -1,67 +1,62 @@
 package de.tum.www1.orion.connector.client
 
 import com.intellij.openapi.project.Project
-import de.tum.www1.orion.connector.client.JavaScriptConnector.JavaScriptFunction.*
```

Comment:
Probably because when I move the JavaScriptFunction class out of JavaScriptConnector, the IDE optimizes the imports automatically. As for why I do that, it's just taste I guess since in kotlin you can write multiple classes and interfaces in the same file. I will change it back if that's what you prefer.